### DISCUSSION TITLE
Search bar disappeared after replacing `react-native-screens/native-stack` with `@react-navigation/native-stack` — is it a bug?

### DISCUSSION THREAD

**Discussion by SimpleCreations** [software-mansion/react-native-screens#1071] [Q&A] [answered]
Hello!

3.6.0 got released today with a warning when importing from `react-native-screens/native-stack` package.
I've replaced the import with `@react-navigation/native-stack`, but after running the app, it has no search bar anymore.
Additionally, I get a TypeScript error because it can't find the `searchBar` screen option in the type definitions.

I used the minimal repro from this issue: #1027 (the code is the same, the library has been upgraded to 3.6.0).

**Comment by kacperkapusciak** [comment, marked as answer]
Hi @SimpleCreations,
many props were renamed between versions including `searchBar`. In `@react-navigation/native-stack` this prop is called `headerSearchBarOptions`.

[native-stack documentation in React Navigation v6](https://reactnavigation.org/docs/native-stack-navigator#headersearchbaroptions)

Cheers!

**Reply by SimpleCreations** [reply]
Thank you, worked like a charm!
Although couldn't find an equivalent of `disableBackButtonMenu: true` in `@react-navigation/native-stack`. Is it available?

**Reply by kacperkapusciak** [reply]
@SimpleCreations 
We've missed it! Thanks for pointing that out. It will be probably available in the next release of `@react-navigation/native-stack` 
https://github.com/react-navigation/react-navigation/pull/9881
https://github.com/react-navigation/react-navigation.github.io/pull/1049
